packages/components/src/AboutDialog/AboutDialog.component.js

```diff
@@ -9,19 +9,11 @@ import theme from './AboutDialog.scss';
 
 import I18N_DOMAIN_COMPONENTS from '../constants';
 
-function Text({ text, loading }) {
-	return (
-		<div>
-			{loading ? <Skeleton type={Skeleton.TYPES.text} size={Skeleton.SIZES.large} /> : text}
-		</div>
-	);
+function Text({ text, loading, size = Skeleton.SIZES.medium }) {
+	return <div>{loading ? <Skeleton type={Skeleton.TYPES.text} size={size} /> : text}</div>;
 }
 
-function Table({ services, t }) {
-	if (!services || !services.length) {
```

Comment:
are you sure we can delete this test? for exemple if there is a problem with the fetch method (saga) will be not defined

Reply:
Ok I revert